app/controllers/katello/api/v2/content_views_controller.rb

```diff
@@ -164,7 +170,7 @@ def remove
       options[:content_view_versions] = versions
       options[:content_view_environments] = cv_envs
 
-      task = async_task(::Actions::Katello::ContentView::Remove, @content_view, options)
+      task = params[:force_destory_cv] ? async_task(::Actions::Katello::ContentView::ForceDestroy, @content_view, options) : async_task(::Actions::Katello::ContentView::Remove, @content_view, options)
```

Comment:
I think i'd like it more if :destroy was an option into the Remove action, rather than trying to string them like this.  A few more comments might make that simpler